addons/account/models/res_config_settings.py

```diff
@@ -145,6 +145,9 @@ class ResConfigSettings(models.TransientModel):
     # Technical field to hide country specific fields from accounting configuration
     country_code = fields.Char(related='company_id.account_fiscal_country_id.code', readonly=True)
 
+    # Storno Accounting
+    storno_accounting = fields.Boolean(string="Storno accounting", readonly=False, related='company_id.storno_accounting')
```

Comment:
account_storno

Reply:
`storno_accounting` field should be renamed to `account_storno` ? Same goes for the field in `company` ?